### ISSUE TITLE
sort by colums

### ISSUE THREAD

**Issue by magicmana** [DeckerMMIV/FarmSim_Mod_Glance#20]
hello,
greats job, but it is possible  to add a possibility to  select  the  colums to sort option ?
for sample i using 3 combine , 3 transborder , and 5 truck with trailer for harvest , and should be fine to have a sort by % fill
regards

**Comment by DeckerMMIV** [comment]
Hi magicmana,

I have tweaked the script-code, to be able to 'sort by % fill'.
Version 1.0.8.47 now has a new XML-attribute which specifies if the vehicle-list should be sorted by the 'fill percent'.

Create a new Glance_Config.XML file, or add the `sortByFillPct="..."` XML-attribute to the XML-tag `vehiclesColumnOrder`:
```
<vehiclesColumnOrder sortByFillPct="false">
```

**Comment by magicmana** [comment]
wahoo quickly dev...
greats tks

**Comment by DeckerMMIV** [comment]
Yeah, well. - This feature at least did not require any DLCs to make ;-)
But for now, only 'fill-percent' and 'VeGS-group/pos' are supported for sorting the list.

**Comment by Vanquish081** [comment]
Where is the file Glance_Config.XML?? I don't see it in folder mods neither FarmingSimulator2017

Edit:
In modsSettings... sorry xD
